newEvaluation.html

```diff
@@ -69,7 +70,7 @@
 	<!-- Section5 -->
 	<section>
 		<h5>Workflow, Leadership & Soft Skills</h5>
```

Comment:
Modern browsers have pretty good mechanisms to try to infer what you mean, even if the actual markup isn't 100% valid. In this case, the issue stems from the fact that in HTML, `&` is a reserved character, as it denotes the start of an HTML entity. In order to include a literal "&" in your page, you'd need to encode it as `&amp;` - the HTML entity denoting the ampersand character, "&".